docs/PARTial_Programming_Guide.md

```diff
@@ -36,17 +36,18 @@ PART is continually being upgraded and at this time includes assessment tools th
 * *Spatial Release from Speech on Speech Masking*
 * *Informational Masking* — Multiple Burst Paradigm
 
-At this time, only a single default Battery is included (‘Listen Battery v2’), which presents limited instructions and examples and includes three of the assessments described above: Binaural Frequency Modulation Detection, Temporal Envelope Gap Discrimination, and Spectrotemporal Modulation Detection. For references related to the assessments provided, see the references in **Section Six**.
+At this time, only a single default Battery is included, which presents limited instructions and examples and includes three of the assessments described above: Binaural Frequency Modulation Detection, Temporal Envelope Gap Discrimination, and Spectrotemporal Modulation Detection. For references related to the assessments provided, see the references in **Section Six**.
 
-A recent modification to reduce the download time is that the CRM sentences and TEN maskers have been removed from the app and are now available as optional downloads. Unless you know you won’t need them, downloading them is recommended to avoid errors with batteries that need them. The best practice is, immediately after installing PART, to go to the Modules screen and download all optional modules. A screenshot of where the Modules tab can be found on the Settings screen is shown below. To get to the settings screen, tap the Settings button on the main screen, shown in **Figure 1a**.
-
-![Figure 1a. PART Start Screen](Images/PART_StartScreen.png)  
+A recent modification to reduce the download time is that Study Batteries, Server Code Batteries, CRM English Talkers, CRM Spanish Talkers, TENoise, Dichotic Sentences English, Dichotic Sentences Spanish, Sequence Digits English, Sequence Digits Spanish, ACT Audio and Images, AVDAT English Audio, AVDAT Spanish Audio, Letter Number Audio, and ER40 Images are now available as optional downloads. To access them, immediately after installing PART, to go to the Modules screen and download all optional modules. A screenshot of where the Modules tab can be found on the Settings screen is shown below. To get to the settings screen, tap the Settings button on the main screen, shown in **Figure 1a**.
```

Comment:
Study batteries and server code batteries do not need to be downloaded and actually cannot be without knowing the name of the batteries you wish to download. These are different from modules, and are used to download specific batteries. They shouldn't be listed here.

Perhaps this wording can be changed to just:

In order to reduce the download time of PART, several modules are available as optional downloads within the app. To access them, go to the Modules screen and download the desired modules. It is considered best practice to download all optional modules immediately after installing PART. Note that Standard Batteries, Study Batteries, and Server Code Batteries are not modules and instead features for downloading specific batteries.